src/cli/generate.ts

```diff
@@ -50,15 +60,60 @@ const argv = yargs.options({
       'Define whether the generated files should be grouped by the @group tag on the top level classes.' +
       'If set to true, a sub directory will be created per group inside of the specified target directory.',
   },
+  indexOnly: {
+    type: 'boolean',
+    default: false,
+    describe:
+      'Defines whether only the index file should be  generated.',
```

Comment:
Minor nitpicky comment: There are 2 spaces in between "be" and "generated"

Reply:
Fixed